modules/sharedaddy/sharing.js

```diff
@@ -267,6 +267,8 @@ var updateLinkedInCount = function( data ) {
 				} );
 				$more_sharing_buttons.data( 'timer', false );
 			} );
+		} else {
+			$( 'body' ).addClass( 'jp-sharing-input-touch' );
```

Comment:
It'd probably be a bit more performant to use `$( document.body )` here, instead of passing `'body'` through the selector engine.

https://developer.mozilla.org/en-US/docs/Web/API/Document/body
https://www.sitepoint.com/jquery-body-on-document-on/